common/io/include/claragenomics/io/fasta_parser.hpp

```diff
@@ -62,7 +54,9 @@ class FastaParser
 /// \param shuffle Enables shuffling reads
 ///
 /// \return A unique pointer to a constructed parser object.
-std::unique_ptr<FastaParser> create_kseq_fasta_parser(const std::string& fasta_file, int min_sequence_length = 0, bool shuffle = true);
+std::unique_ptr<FastaParser> create_kseq_fasta_parser(const std::string& fasta_file,
+                                                      const number_of_basepairs_t min_sequence_length = 0,
```

Comment:
I wouldn't put `const` in the function declaration for call-by-value parameters. It is call-by-value, so the values will be copied - that's the semantic of this function. The constness is an implementation detail, so I would add it only into the implementation of the function. The compiler should fine with this difference.

Reply:
Cool, I didn't know that. Thanks!
Done

Reply:
sure, you're welcome. :)